src/schema/posts.ts

```diff
@@ -508,6 +509,24 @@ export const typeDefs = /* GraphQL */ `
       id: ID!
     ): EmptyResponse @auth
 
+    """
+    Create private post in source
+    """
+    privatePost(
```

Comment:
I find this name a bit confusing. Can you please come up with a new name?
It should also be phrased as an action because it's a mutation (i.e. submitExternalLink)

Reply:
Totally, should we then also consider renaming `sharePost`? or is that explicit enough for now.

Reply:
Renaming it will break production but I find it explicit enough `share` is the action and `post` is the entity you share. I get the confusion though because we have a share post type as well